demo/src/main/java/com/example/DemoApplication.java

```diff
@@ -32,6 +32,9 @@ public void run(String... arg0) throws Exception {
 
 				userRepository.findAll().stream().forEach(user -> System.out.println(user));
 
+				// Intentional bug
+				String str = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "str" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "str". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1854)